ZAPD/CrashHandler.h

```diff
@@ -0,0 +1,21 @@
+#include <array>
+#include <csignal>
+#include <cstdio>
+#include <cstdlib>
+#include <ctime>
+
+#if defined(__linux__)
+#include <cxxabi.h>  // for __cxa_demangle
+#include <dlfcn.h>   // for dladdr
+#include <execinfo.h>
+#include <unistd.h>
```

Comment:
Again, most of these are POSIX rather than Linux-specific, and are perfectly available on Mac, for example.